examples/01_plotting/plot_3d_map_to_surface_projection.py

```diff
@@ -280,13 +280,15 @@
 # See :ref:`interactive-surface-plotting` for more details.
 from nilearn.plotting import view_surf
 
+engine = "niivue"
 view = view_surf(
     surf_mesh=fsaverage_meshes["inflated"],
     surf_map=surface_image,
-    threshold="90%",
+    # threshold="90%",
```

Comment:
Something's wrong with the thresholding, so had to comment it out to display a legible brain